### PR TITLE
Bugfix/pro 1289 notification template allow links

### PR DESCRIPTION
**Pull request by Tschuppi81**
Feriennet: Allow links in notification template

TYPE: Bugfix
LINK: pro-1289

### CODE REVIEW

File: src/onegov/feriennet/models/notification_template.py [hunk 1]
```diff
@@ -145,9 +155,16 @@ def render(self, text: str) -> Markup:
         if len(paragraphs) <= 1:
             result = text
         else:
-            result = Markup('\n').join(as_paragraphs(text))
+            result = '\n'.join(paragraphs)
 
-        return self.expand_storage_links(result)
+        result = self.expand_storage_links(Markup(result))
+
+        return clean(
```
Comment (Tschuppi81) [inline src/onegov/feriennet/models/notification_template.py:162]: I thought about using `linkify` from [core/utils.py](https://github.com/OneGov/onegov-cloud/blob/master/src/onegov/core/utils.py#L406) but it does not allow <p> and <br>. I could easily add them but not sure if its a good idea.

File: src/onegov/feriennet/models/notification_template.py [hunk 2]
```diff
@@ -134,8 +119,33 @@ def bind(
...
+    def render(self, text: str) -> str:
+        """
+        Escapes and renders the given text, replacing all tokens with the
+        result of the bound methods and 'linkifying' the result.
+
+        :param text:
+        :return: clean text
+        """
+
+        def as_paragraphs(text: str) -> Iterator[str]:
```
Comment (Tschuppi81) [inline src/onegov/feriennet/models/notification_template.py:131]: NOTE: I moved `as_paragraphs` inside `render` method as I reverted using `Markup` and `as_paragraphs` does not return safe html code. I finally use `bleach.clean` to make it safe.

Comment (Daverball) [reply src/onegov/feriennet/models/notification_template.py:131]: I'd go back to the safe `as_paragraphs` function, there's still plenty of ways to add links this way.

**Comment by Tschuppi81** [comment]
New solution found, closing

**Pull request closed without merging**